pipelines/update_raster_stats.py

```diff
@@ -0,0 +1,148 @@
+from datetime import datetime
+
+import pandas as pd
+import xarray as xr
+from tqdm.auto import tqdm
+
+from src.constants import ISO3S
+from src.datasources import codab
+from src.utils import blob, database
+
+
+def get_existing_stats_dates(iso3: str, engine) -> list:
+    query = f"""
+    SELECT DISTINCT valid_date
+    FROM app.floodscan_exposure
+    WHERE iso3 = '{iso3.upper()}'
+    ORDER BY valid_date
+    """
+    df_unique_dates = pd.read_sql(query, con=engine)
+    df_unique_dates["valid_date"] = pd.to_datetime(
+        df_unique_dates["valid_date"]
+    )
+    return df_unique_dates["valid_date"].to_list()
+
+
+if __name__ == "__main__":
+    engine = database.get_engine(stage="dev")
+    clobber = False
+    verbose = False
```

Comment:
Not critical at this stage, but I like using the `logging` module with different log levels to determine how verbose we want our logs to be. This way we have a bit more control than the binary on/off.

Reply:
Yes great call, will add that later #11